dashboard/app/controllers/dynamic_config_controller.rb

```diff
@@ -65,4 +65,54 @@ def gatekeeper_set
     flash[:notice] = "Updated successfully! Remember your changes take 30 seconds to go into effect, so don't expect to see the changes immediately on this page."
     redirect_to action: :gatekeeper_show, feature: feature
   end
+
+  def dcdo_show
+    authorize! :read, :reports
+    DCDO.refresh
+
+    @dcdo_hsh = DCDO.to_h
+  end
+
+  def dcdo_set
+    authorize! :read, :reports
+    DCDO.refresh
+
+    params.require([:key, :data_type])
+    key = params[:key]
+    raw_value = params[:value]
+    data_type = params[:data_type]
+
+    begin
+      value =
+        case data_type
+          when "Integer"
+            Integer(raw_value)
+          when "Float"
+            Float(raw_value)
+          when "Boolean"
+            raw_value.to_bool
+          when "String"
+            raw_value
+          else
+            new_value = JSON.parse(raw_value)
```

Comment:
Should we use `oj` to parse for consistency with how the DCDO storage adapters serialize and load values as they fetch/write? We’d have to figure out the equivalent `oj` Errors to rescue down below, though.